scripts/py_matter_yamltests/matter_yamltests/websocket_runner.py

```diff
@@ -24,6 +27,10 @@
 
 _KEEP_ALIVE_TIMEOUT_IN_SECONDS = 120
 _MAX_MESSAGE_SIZE_IN_BYTES = 10485760  # 10 MB
+_CONNECT_MAX_RETRIES_DEFAULT = 3
```

Comment:
This used to be 4 before https://github.com/project-chip/connectedhomeip/commit/db067cb69251254cf276c293f15dfeda909106e9, no?

Reply:
Indeed.